app/backend/src/auth/auth.service.ts

```diff
@@ -38,7 +38,11 @@ export class AuthService {
       newuser.firstConnection = true;
       return newuser;
     }
-    logger.debug("Returning user");
+    logger.debug("Returning user email: ", data.email);
+    await this.prisma.user.update({
```

Comment:
This is already done in the chat gateway

Reply:
this is for log in, it needs to be done when user reconnects, to be set to ONLINE.

Reply:
Still done in the chat gateway